bootstrap/config.aws-aws.tpl

```diff
@@ -15,65 +21,18 @@
           ]
         },
         "Instance": {
-          "Plugin": "instance-aws",
-          "Properties": {
-            "RunInstancesInput": {
-              "ImageId": "{{ ref "/aws/amiid" }}",
-              "InstanceType": "{{ ref "/aws/instancetype" }}",
-              "KeyName": "{{ ref "/aws/keyname" }}",
-              "SubnetId": "{{ ref "/aws/subnetid" }}",
-              {{ if ref "/aws/instanceprofile" }}"IamInstanceProfile": {
-                "Name": "{{ ref "/aws/instanceprofile" }}"
-              },{{ end }}
-              "SecurityGroupIds": [ "{{ ref "/aws/securitygroupid" }}" ]
-            },
-            "Tags": {
-              "Name": "{{ ref "/aws/stackname" }}-manager",
-              "Deployment": "Infrakit",
-              "Role" : "manager"
-            }
-          }
+          {{ include "instace-aws.tpl" "config.aws-aws.tpl" }}
```

Comment:
Originally, it was for giving context to the template file so that local variables could be passed to it before it was rendered. Since we are using globals now, this is obsolete.